hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This patch fixes a very rare overflow bug (:issue:`1748`) in
+:func:`~hypothesis.strategies.complex_numbers`.  Floating-point math is *hard*.
```

Comment:
This entry amuses me but I think we should probably not use it as is. ` Floating-point math is *hard*.` isn't actually a very helpful changelog entry. 

Also more importantly we should probably say what you would have seen if triggering this bug (an `InvalidArgument` error)